workshop_attendance.txt

```diff
@@ -1,4 +0,0 @@
-# Please enter your name and your github handle, each in a new line.
-
-Blanca Mancilla
-@bluciam
```

Comment:
Yes, please delete that pull request.  You should have received a second
pull request from me with the file kshred_attendance.txt (or something to
that effect.)

Thanks,
Kyla

2015-10-09 12:20 GMT-04:00 Blanca Mancilla <notifications@github.com>:

> In workshop_attendance.txt
> <https://github.com/bluciam/helloworld_pyladiesMTL/pull/39#discussion_r41648910>
> :
>
> > @@ -1,4 +0,0 @@
> > -# Please enter your name and your github handle, each in a new line.
> > -
> > -Blanca Mancilla
> > -@bluciam
>
> Should be delete this pull requests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bluciam/helloworld_pyladiesMTL/pull/39/files#r41648910>
> .
>



-- 
molt.bandcamp.com